packages/fluentui/digest/src/webpack.config.ts

```diff
@@ -1,7 +1,31 @@
+import { registerTsPaths, rules } from '@fluentui/scripts-storybook';
```

Comment:
this should be probably refactored in the future to standalone `scripts-webpack-utils` package, as this crosses domain between v0,v8,v9 + the package is primarily for storybook purposes (but because sb is coupled with webpack we are in this situation...)